app/controllers/student_flashcards_controller.rb

```diff
@@ -0,0 +1,62 @@
+class StudentFlashcardsController < ApplicationController
+  before_action :find_student_flashcard, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @student_flashcard_set = StudentFlashcardSet.find(params[:student_flashcard_set_id])
+    @student_flashcards = policy_scope(StudentFlashcard.where(student_flashcard_set: @student_flashcard_set))
+    # @student_flashcard = StudentFlashcard.new
+    authorize @student_flashcard
```

Comment:
you need to authorize @student_flashcard_set, not the @student_flashcard**s** cos that's being verified with the policy scope